cypress/integration/router_spec.js

```diff
@@ -29,13 +29,14 @@ describe('Router Flow', () => {
   it('should display the navigation buttons on the main page', () => {
     cy.get('#doctors-button').should('contain', 'Doctors')
       .get('#mental-health-pros-button').should('contain', 'Mental Health Professionals')
-      .get('#submit-button').should('contain', 'Submit')
+      .get('#submit-button').should('contain', 'Suggest')
```

Comment:
I like this for providing a bit more accuracy for the user about the functionality of the button